views/monitor.hbs

```diff
@@ -1,4 +1,25 @@
 {{!< default }}
+<div id="banner-info" class="section-container">
+  <div id="banner-sections-wrapper">
+    <div id="banner-left">
+      <div class="banner-content">
+        {{> protect_yourself }}
+        </div>
+    </div><!--closes banner-left-->
+    <div id="banner-right">
+      <img id="arrow-divider" src="img/banner_arrow.svg" alt="content divider" />
```

Comment:
We could probably use an empty `alt=""` here, since this alt text doesn't add much (and may confuse screen readers).